pkg/trace/config/config.go

```diff
@@ -141,16 +141,16 @@ type Tag struct {
 // New returns a configuration with the default values.
 func New() *AgentConfig {
 	ctx, cancel := context.WithTimeout(context.Background(), 2*time.Second)
-	isFargate := fargate.IsFargateInstance(ctx)
+	orch := fargate.GetOrchestrator(ctx)
 	cancel()
 	if err := ctx.Err(); err != nil && err != context.Canceled {
-		log.Errorf("Error detecting Fargate instance: %v. Assuming not. If you are in a Fargate instance, this will cause problems.", err)
+		log.Warnf("Failed to get Fargate orchestrator. This may cause issues if you are in a Fargate instance: %v", err)
```

Comment:
I think this should be more than just a warning since it can have some serious implications on APM stats and billing. Why did you change it?

Reply:
It was warning in the other place. I didn't intend to change it.